airbyte-integrations/connectors/destination-postgres/src/test-integration/java/io/airbyte/integrations/destination/postgres/PostgresDestinationAcceptanceTest.java

```diff
@@ -34,7 +34,7 @@ protected String getImageName() {
   @Override
   protected JsonNode getConfig() {
     return Jsons.jsonNode(ImmutableMap.builder()
-        .put("host", db.getHost())
+        .put("host", "host.docker.internal")
```

Comment:
We should trigger postgres acceptance tests to make sure this would not cause any trouble

Reply:
yeah. this is actually intended to _fix_ postgres acceptance tests (seems to not be working), since `base-normalization:integrationTest` now also runs `destination-postgres:integrationTest`